### PR TITLE
Includes run() documentation

### PR DESCRIPTION
**Pull request by ruiconti**
**Proposed changes**:
- Included Action's `run()` documentation for sphinx reference

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: rasa_core_sdk/__init__.py [hunk 1]
```diff
@@ -151,7 +151,20 @@ def name(self):
...
+            dispatcher (CollectingDispatcher): the dispatcher which is used to send
+                messages back to the user. Use ``dipatcher.utter_message()``
+                or any other :class:`rasa_core_sdk.executor.CollectingDispatcher` method.
+            tracker (Tracker): the state tracker for the current
+                user. You can access slot values using
+                ``tracker.get_slot(slot_name)``, the most recent user
+                message is ``tracker.latest_message.text`` and any other  :class:`rasa_core_sdk.Tracker` property.
+            domain (Dict[Text, Any]): the bot's domain
+        Returns:
+            List[Event]: A list of :class:`rasa_core_sdk.events.Event` instances that is returned through the endpoint
```
Comment (Ghostvv) [inline rasa_core_sdk/__init__.py]: `Event`'s are `dict`s in `rasa_core_sdk`, so it returns list of dict

**Comment by akelad** [comment]
@ruiconti would you mind updating your PR?

File: rasa_core_sdk/__init__.py (hunk 1 above)
Comment (ruiconti) [reply rasa_core_sdk/__init__.py]: Done

**Comment by ruiconti** [comment]
@akelad Sure! Sorry for the lateness

File: rasa_core_sdk/__init__.py [hunk 2]
```diff
@@ -151,7 +151,23 @@ def name(self):
...
+                or any other :class:`rasa_core_sdk.executor.CollectingDispatcher`
+                method.
+            tracker (Tracker): the state tracker for the current
+                user. You can access slot values using
+                ``tracker.get_slot(slot_name)``, the most recent user message
+                is ``tracker.latest_message.text`` and any other
+                :class:`rasa_core_sdk.Tracker` property.
+            domain (Dict[Text, Any]): the bot's domain
+        Returns:
+            Dict[Event]: A dictionary of :class:`rasa_core_sdk.events.Event`
```
Comment (akelad) [inline rasa_core_sdk/__init__.py]: @ruiconti sorry, what vova meant was a list of dicts, like in your core PR:
```suggestion
            List[dict]: A dictionary of :class:`rasa_core_sdk.events.Event`
```

**Pull request merged**